test/test.getch.ts

```diff
@@ -133,14 +133,27 @@ describe('🥁 configuration options', () => {
     scope.done();
   });
 
+  it('should merge URL parameters with params option', async () => {
+    const opts = {
+      url: `${url}/?james=beckwith&montgomery=scott`,
+      params: {james: 'kirk'}
+    };
+    const path = '/?james=kirk&montgomery=scott';
+    const scope = nock(url).get(path).reply(200, {});
+    const res = await request(opts);
+    assert.strictEqual(res.status, 200);
+    assert.strictEqual(res.config.url, url + path);
+    scope.done();
+  });
+
   it('should allow overriding the param serializer', async () => {
     const qs = '?oh=HAI';
     const params = {james: 'kirk'};
     const opts: GaxiosOptions = {
       url,
       params,
       paramsSerializer: (ps) => {
-        assert.deepStrictEqual(params, ps);
+        assert.deepEqual(params, ps);
```

Comment:
Why drop the `strict`?

Reply:
We recreate the object in the code with the combination of the params specified in the `url` as well as the `opts.params`, so it's not the original object that gets passed in here.